layouts/shortcodes/cli/influxd-flags.md

```diff
@@ -1,35 +1,39 @@
+|      | `--storage-cache-snapshot-write-cold-duration`              | Length of time at which the engine will snapshot the cache and write it to a new TSM file if the shard hasn't received writes or deletes (default `10m0s`)                                                                                                                 | duration   |                                      |
+|      | `--storage-compact-full-write-cold-duration`                | Duration at which the engine will compact all TSM files in a shard if it hasn't received writes or deletes (default `4h0m0s`)                                                                                                                                              | duration   |                                      |
+|      | `--storage-compact-throughput-burst`                        | Rate limit in bytes per second that TSM compactions can write to disk (default `48 MiB`)                                                                                                                                                                                   | integer    |                                      |
+|      | `--storage-max-concurrent-compactions`                      | Maximum number of concurrent full and level compactions that can run at one time.  A value of 0 results in 50% of runtime. `GOMAXPROCS(0)` used at runtime. Any number greater than 0 limits compactions to that value. This setting does not apply to cache snapshotting. | integer    |                                      |
+|      | `--storage-max-index-log-file-size`                         | Threshold, in bytes, when an index write-ahead log file will compact into an index file. Lower sizes will cause log files to be compacted more quickly and result in lower heap usage at the expense of write throughput (default `1.0 MiB`)                               | integer    |                                      |
+|      | `--storage-series-file-max-concurrent-snapshot-compactions` | Maximum number of concurrent snapshot compactions that can be running at one time across all series partitions in a database                                                                                                                                               | integer    |                                      |
+|      | `--storage-series-id-set-cache-size`                        | Size of the internal cache used in the TSI index to store previously calculated series results                                                                                                                                                                             | integer    |                                      |
```

Comment:
This looks great, however, there are a few additional arguments that have been added since this list was created. I have include the complete list:

```
--storage-cache-max-memory-size Size                            The maximum size a shard's cache can reach before it starts rejecting writes. (default 1.0 GiB)
--storage-cache-snapshot-memory-size Size                       The size at which the engine will snapshot the cache and write it to a TSM file, freeing up memory. (default 25 MiB)
--storage-cache-snapshot-write-cold-duration Duration           The length of time at which the engine will snapshot the cache and write it to a new TSM file if the shard hasn't received writes or deletes. (default 10m0s)
--storage-compact-full-write-cold-duration Duration             The duration at which the engine will compact all TSM files in a shard if it hasn't received a write or delete. (default 4h0m0s)
--storage-compact-throughput-burst Size                         The rate limit in bytes per second that we will allow TSM compactions to write to disk. (default 48 MiB)
--storage-max-concurrent-compactions int                        The maximum number of concurrent full and level compactions that can run at one time.  A value of 0 results in 50% of runtime.GOMAXPROCS(0) used at runtime.  Any number greater than 0 limits compactions to that value.  This setting does not apply to cache snapshotting.
--storage-max-index-log-file-size Size                          The threshold, in bytes, when an index write-ahead log file will compact into an index file. Lower sizes will cause log files to be compacted more quickly and result in lower heap usage at the expense of write throughput. (default 1.0 MiB)
--storage-retention-check-interval Duration                     The interval of time when retention policy enforcement checks run. (default 30m0s)
--storage-series-file-max-concurrent-snapshot-compactions int   The maximum number of concurrent snapshot compactions that can be running at one time across all series partitions in a database.
--storage-series-id-set-cache-size int                          The size of the internal cache used in the TSI index to store previously calculated series results.
--storage-shard-precreator-advance-period Duration              The default period ahead of the endtime of a shard group that its successor group is created. (default 30m0s)
--storage-shard-precreator-check-interval Duration              The interval of time when the check to pre-create new shards runs. (default 10m0s)
--storage-tsm-use-madv-willneed                                 Controls whether we hint to the kernel that we intend to page in mmap'd sections of TSM files.
--storage-validate-keys                                         Validates incoming writes to ensure keys only have valid unicode characters.
--storage-wal-fsync-delay Duration                              The amount of time that a write will wait before fsyncing. A duration greater than 0 can be used to b
```

Reply:
@stuartcarnie 

> A duration greater than 0 can be used to b

I think the end of this sentence is missing?

Reply:
Apologies, a copy error:

```
--storage-wal-fsync-delay Duration                              The amount of time that a write will wait before fsyncing. A duration greater than 0 can be used to batch up multiple fsync calls. This is useful for slower disks or when WAL write contention is seen. (default 0s)
```